test/inductor/test_flex_attention.py

```diff
@@ -2977,324 +2993,87 @@ def mod(b, h, q, kv):
         torch.compile(create_block_mask)(mod, None, None, 1023, 1023, device=device)
         self.run_test_with_call(
             lambda q, k, v: flex_attention(q, k, v, block_mask=block_mask),
+            torch.float16,
+            device,
             Q_S=1023,
             KV_S=1023,
-            device=device,
         )
 
     @supported_platform
-    @unittest.skipIf(SKIP_UT_ON_CPU, "Skip on CPU as not supported")
-    def test_head_bias_req_grad(self):
```

Comment:
curious why moving these tests below? It might be better to minimize the change

Reply:
Should have been down there so fixing while I am here